app/models/governing-body.ts

```diff
@@ -11,6 +16,18 @@ export default class GoverningBodyModel extends Model {
   })
   declare administrativeUnit: AdministrativeUnitModel;
 
-  @belongsTo('session', { async: true, inverse: 'governingBody' })
-  declare session: AsyncBelongsTo<SessionModel>;
+  @hasMany('session', { async: true, inverse: 'governingBody' })
+  declare sessions: AsyncHasMany<SessionModel>;
+
+  @belongsTo('governing-body', {
+    async: false,
+    inverse: 'hasTimeSpecializations',
+  })
+  declare isTimeSpecializationOf: GoverningBodyModel;
```

Comment:
**Question:** Can you clarify what timeSpecialization is for/refers to?

Reply:
There are 2 types of governing bodies:
- the abstraction (undated)
- the time specified

This is described in the mandatendatabank specification: https://data.vlaanderen.be/doc/applicatieprofiel/mandatendatabank/html/overview.jpg

From a Session, we are supposed to go to `governingBody.isTimeSpecializationOf.administrativeUnit`. 

I discover it last Friday (https://chat.semte.ch/group/X8kirHFphoFXfBKXZ?msg=jxCt2wxns58ssKpuP)